internal/cluster/cluster.go

```diff
@@ -263,8 +263,10 @@ func (m *Manager) reportValidationStatusChanged(ctx context.Context, c *common.C
 	for vCategory, vRes := range newValidationRes {
 		for _, v := range vRes {
 			if currentStatus, ok := m.getValidationStatus(currentValidationRes, vCategory, v.ID); ok {
-				if v.Status == ValidationFailure && currentStatus == ValidationSuccess {
-					m.metricAPI.ClusterValidationChanged(models.ClusterValidationID(v.ID))
+				if v.Status == ValidationFailure && currentStatus != ValidationFailure {
+					if currentStatus == ValidationSuccess {
```

Comment:
Why just success? why not also pending and error?

Reply:
The metric name is:
`assisted_installer_cluster_validation_failed_after_success_before_installation`